src/main/resources/application.properties

```
@@ -0,0 +1,13 @@
+spring.datasource.url= ENC(wVfTRIMaMxRW0CszoDjaLzH1BHdNREhuiWkqW/EYR0QXXN2maoIp02ikHFoATq+V)
+spring.datasource.username= ENC(Z47dmZEznIK9YAUsk6BVjg==)
+spring.datasource.password= ENC(GBton3uY9bLBTIc76KYNGQ==)
```

It's better here to be simple. Use environment variables directly, which we can set with `.env` files (**to be added to `.gitignore`**. but we can have a `.env.example` commited).

Please see https://12factor.net for more details about good practice for deployment. This would be critical for #4.

```suggestion
spring.datasource.url=jdbc:postgres://${POSTGRES_HOST}:${POSTGRES_PORT}/${POSTGRES_DB}
spring.datasource.username=${POSTGRES_USERNAME}
spring.datasource.password=${POSTGRES_PASSWORD}
```